src/auth/src/token.rs

```diff
@@ -23,7 +23,7 @@ pub struct Token {
 
     /// The instant at which the token expires. If `None`, the token does not
     /// expire (or its expiration is unknown).
-    pub expires_at: Option<std::time::Instant>,
+    pub expires_at: Option<time::OffsetDateTime>,
```

Comment:
And then I noticed that the time crate is `0.3.*` and therefore not suitable for a public interface.  Feel free to merge as-is, just open a tracking bug.

@codyoss any thoughts?

Reply:
Filed https://github.com/googleapis/google-cloud-rust/issues/387